.swiftlint.yml

```diff
@@ -6,6 +6,8 @@ excluded:
 analyzer_rules:
   - unused_declaration
   - unused_import
+disabled_rules:
+  - redundant_optional_initialization
```

Comment:
Temporarily disabled this rule. See #3998.

Reply:
Adding another constructor to the property wrapper allowed to avoid the explicit initialization with `nil`. Thus, the rule is now active again.